XU-TechnicalPaper.md

```diff
@@ -102,17 +102,60 @@ Payment channel atomic swap technology is fairly new and was just recently succe
+
+Order matching systems ('trade engines') of major digital asset exchanges differentiate between two participants: the taker and the maker. A maker is someone who submits an order which can't be matched immediately and joins a pool of unmatched orders. A taker on the other hand issues an order which can immediately be matched with an existing (maker) order upon entering the pool. Makers, are sometimes also called 'liquidity providers' and play an important role in stabilizing markets and thus are rewarded with lower or 0% fees on centralized exchanges, which is why we chose a similar model for XU. In short, a taker -> pays the maker as an incentive to provide liquidity. Details below.
+
+Similarly, we have two types of participants in the XU's DOB for each trade:
+1. The Maker, the liquidity provider propagating a order, which joins a pool of unmatched orders. Usually in the form of a limit order.
+2. The Taker, the impatient buyer or seller filling above limit orders. Usually in the form of a market order.
+
+A description of how we realize above goals:
+
+1. Minimize latency of order updates
+For orders, the DOB protocol strictly follows the first come, first served principle, which remains fully compatible with how order book systems of exchanges work. To get the best achievable latency between two nodes which intend to receive order updates from each other, we require an open connection between two nodes on Internet Protocol level. From a XU network topology point of view we require a direct connection without intermediary hop with all nodes a node wants to receive order information from. Transport Layer design choices using socket connections and [efficient congestion control mechanisms](https://github.com/google/bbr) are currently in the works.
+
+2. Single point of execution 
+There is one single point of execution for each order: the maker. The maker decides on which incoming order gets to fill the order. All orders contain payment information (e.g. a lightning invoice). Lightning is defined in a way (See (BOLT #11))[https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md], that an invoice can only be paid by the first one to successfully send a payment to an invoice issuer, in our case the maker. Also lightning takes care that a taker has a payment channel path with sufficient volume for a specific order. If the incoming order can't be filled because it was already filled by someone else or canceled, the makers responsibility is to send the funds back in a timely manner, otherwise there will be a punishment using its staked XUC (details tbd). Worst-case the takers payment is locked for the the time as specified in the Atomic Swap HTLC and the maker gets punished. Under no circumstances a permanent loss of funds can occur.
+
+Sidenote: Transferring order execution authority to a third party, in particular to a decentralized consensus, was considered and deemed not feasible for the following reasons:
```

Comment:
We should note that third parties can't truly "control" order execution, only the creator of an invoice (and the accompanying order) can "execute" it by releasing the preimage.

Reply:
Tried to add that. Check again.